src/main/java/org/candlepin/sync/Exporter.java

```diff
@@ -600,33 +600,28 @@ private void exportDistributorVersions(File baseDir) throws IOException {
     }
 
     private void exportContentDeliveryNetworks(File baseDir) throws IOException {
-        ResultIterator<Cdn> iterator = this.cdnCurator.listAll().iterate();
+        Iterator<Cdn> iterator = this.cdnCurator.listAll().iterator();
 
-        try {
-            if (iterator.hasNext()) {
-                File cdnDir = new File(baseDir.getCanonicalPath(), "content_delivery_network");
-                cdnDir.mkdir();
-
-                while (iterator.hasNext()) {
-                    Cdn cdn = iterator.next();
-                    log.debug("Exporting CDN: {}", cdn.getName());
-
-                    FileWriter writer = null;
-                    try {
-                        File file = new File(cdnDir.getCanonicalPath(), cdn.getLabel() + ".json");
-                        writer = new FileWriter(file);
-                        cdnExporter.export(mapper, writer, cdn);
-                    }
-                    finally {
-                        if (writer != null) {
-                            writer.close();
-                        }
+        if (iterator.hasNext()) {
```

Comment:
Since we have to hit this method, let's take the time to clean it up. We can make use of try-with-resources to avoid the finally blocks, and it's no longer necessary to explicitly close the outer iterator since we aren't worried about the cursor hanging open. In fact, an iterator probably isn't even the most effective way to do this.

```
List<Cdn> cdns = this.cdnCurator.listAll();

if (cdns != null && !cdns.isEmpty()) {
  // create cdn directory

  for (CDN cdn : cdns) {
    File cdnFile = ...
    try (FileWriter writer = ...) {


    }
  }
}
```